HttpClient/Client.cs

```diff
@@ -0,0 +1,58 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Sockets;
+using System.Text;
+
+namespace HttpClient
+{
+	public class Client
+	{
+		public static string Get(string host, Dictionary<string, string> headers)
+		{
+			var hostEntries = Dns.GetHostEntry(host);
+			var ipAddress = hostEntries.AddressList[0];
+			const int port = 80;
+			var ipEndPoint = new IPEndPoint(ipAddress, port);
+			using var socket = new Socket(AddressFamily.InterNetwork, SocketType.Stream, ProtocolType.Tcp);
+
+			socket.Connect(ipEndPoint);
+
+			//Cache-Control: no-cache
+
+			IEnumerable<string> parsedHeaders = headers.Select(header => header.Key + ":" + header.Value + "\r\n");
```

Comment:
technically this is the opposite of parsed. There's nothing wrong with the code but the variable name is misleading. It's serializing headers